docs/helm/examples/alf_license.md

```diff
@@ -0,0 +1,44 @@
+# Deploy Alfresco Content Services with the ability to manage licenses
+
+Alfresco content repository Docker images come with an embeded license which last only 2 days. If you've built a custom image, it may have a one year license but in order to change it you would need to biuld a new image.
```

Comment:
```suggestion
Alfresco content repository Docker images come with an embedded license which last only 2 days. If you've built a custom image, it may have a one year license but in order to change it you would need to build a new image.
```